data/Kevinkp09.json

```diff
@@ -19,5 +27,16 @@
       "url": "https://github.com/Kevinkp09",
       "icon": "FaGithub"
     }
-  ]
+  ],
+  "Milestones": [
```

Comment:
should be in lowercase **"milestones"**

Reply:
But on Linkfree website all members have "Milestones" instead of "milestones". So can you please tell me the reason why you suggested this. Thanks for reviewing it😀

Reply:
sure! In simple words, that is a front-end text that you see. Here, it is a key of the `JSON` object. 

For instance, consider a box with the title "Name" and the names of different users below it. This title can be a `<h1> tag` like `<h1>Name</h1>`. Now suppose the names that we want to keep below it are in an object (key-value) like `"name" : "Kevin"`. To show this name we have to use the key `name` and not the title `Name`. See below:

```
<h1>Name</h1> // here the title is hard-coded as "Name"
<p>{json.name}</p> // here we are getting the value from the JSON object using a dot and the key
```
Similarly, "**Milestones**" is a tab in the *linkfree* website that you see in the front end. Under that tab, we may have different milestones that will be present inside the "**milestones**" key from the `.json` file that you have created (notice the key is in lowercase).

Now, this will make more sense when you look at the "**My Links**" tab on the website. There you can see different links that you have provided in your `.json` file under the "**links**" key.

Please note that as for contributing your profile, you don't have to worry about the implementation. You just have to create the valid `.json` file and the rest will be done automatically for you.

Reply:
Also, you can test your changes by yourself in the [LinkFree Playground](https://linkfree.io/playground)